lib/stacks/lib/assets/setup-instance-store-volumes.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
@frbrkoala I don't understand the purpose of this file but figured it was better to leave old wisdom in. What's going on with this `instance-store` business and why would we need to have this as a separate script?

Reply:
it looks like a generic script to ensure an additional data volume (from env var DATA_VOLUME_ID) has a valid fileysystem and is present in `/etc/fstab` so it will be available on reboots. 

failing this, there would need to be some logic in the CDK code to mount additional volumes via user-data.

Reply:
Got it. I think this script is already the fail safe from [stacks/.../node.sh#L124](https://github.com/stacks-network/aws-blockchain-node-runners/blob/feat/stacks-node/lib/stacks/lib/assets/user-data/node.sh#L124) which will always run for the `instance-store` volume type. (Taken from the Solana lib)

It's not in the ethereum node startup [ethereum/.../node.sh#L201-L232](https://github.com/stacks-network/aws-blockchain-node-runners/blob/feat/stacks-node/lib/ethereum/lib/assets/user-data/node.sh#L201-L232) but the Ethereum node **does** allow the `instance-store` in the config [here](https://github.com/stacks-network/aws-blockchain-node-runners/blob/feat/stacks-node/lib/ethereum/lib/config/ethConfig.ts#L14-L15), maybe this method is more robust?

Reply:
The script in this PR is a bit more robust vs the other examples you've shared. 
better error checking and better at preventing duplication in fstab. the cronjob still rubs me the wrong way, but changing that to a systemd timer/oneshot unit is debatable.

Reply:
The problem with Instance Store is that it is ephemeral, meaning that is the instance has been stopped/started for some reason, the attached instance store discs have to be re-initialized. This script checks if the instance store discs are still available after the reboot and, if not, re-initializes them. As mentioned, the use of cronjob is just because I'm more familiar with it and it seems to work well. Open for alternative implementations.